SPECS/bind/bind.spec

```diff
@@ -412,7 +412,7 @@ fi;
 %dir /run/named
 
 %files libs
-%{_libdir}/*so.*
+%{_libdir}/*-%{version}*.so
```

Comment:
The `*.so` files are more common to the `*-devel` subpackages. What happened, that we moved these around? I'm not sure, if this is correct.

Reply:
[As of bind 9.16.12](http://gitlab.isc.org/isc-projects/bind9/-/blob/v9_16/CHANGES#L372), the installed libraries have had their sonames changed to include the version number. This versioning replaces the versioning after the `.so`. I believe this is to discourage third party applications from using the libraries.

[Fedora also packages the libraries in this way.](https://src.fedoraproject.org/rpms/bind/blob/rawhide/f/bind.spec#_963)